src/main/java/net/openhft/chronicle/network/cluster/ClusterContext.java

```diff
@@ -226,7 +226,10 @@ public void cluster(Cluster<T, C> cluster) {
         this.cluster = cluster;
     }
 
-    protected abstract void defaults();
+    protected void defaults() {
+        pauserSupplier = DEFAULT_PAUSER_MODE;
```

Comment:
these are the same defaults that were in field initialiser - now just called here, where they can be easily overridden